src/workflow/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ add_library(openstudio_workflow
   Timer.cpp
 )
 
-target_link_libraries(openstudio_workflow PRIVATE openstudiolib)
+target_link_libraries(openstudio_workflow PUBLIC openstudiolib)
```

Comment:
This one looks slightly problematic to me, especially since it appears the issue is in the workflow_test

```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error	LNK2005	"public: virtual __cdecl openstudio::IdfObject::~IdfObject(void)" (??1IdfObject@openstudio@@UEAA@XZ) already defined in RunPreProcessMonthlyReports_GTest.obj	openstudio_workflow_tests	C:\repos\os\debug\src\workflow\openstudiolib.lib(openstudiolib.dll)	1	
```

Reply:
It's possible that this might work instead, I'll try it out:

-target_link_libraries(openstudio_workflow PUBLIC openstudiolib)
+target_link_libraries(openstudio_workflow PRIVATE openstudiolib)

 if(BUILD_TESTING)
   set(openstudio_workflow_test_depends
+    openstudiolib
     openstudio_workflow
-    boost::boost # Maybe at some point replace with openstudiolib more simply
+    boost::boost
     fmt::fmt
   )

Reply:
Yup that works, I can push if you prefer.  However, that requires anyone who links openstudio_workflow to also link openstudiolib which is what the PUBLIC link option does.

Reply:
I don't think anyone should link to openstudio_workflow.

Reply:
Public is fine to me. You guys understand the implications.